swagger.yaml

```diff
@@ -615,10 +618,10 @@ paths:
         - "application/json"
       parameters:
         - in: path
-          name: name
+          name: id
           type: string
           required: true
-          description: the groups name
+          description: the groups ID
         - in: path
           name: user_id
           type: string
```

Comment:
LIne 636 below still references group `name` rather than `id`:

Current:
```
description: "Invalid group name or user id provided"
```
Expected:
```
description: "Invalid group id or user id provided"
```